python/tvm/relax/frontend/onnx_frontend.py

```diff
@@ -223,23 +224,63 @@ def _impl_v13(cls, bb, inputs, attr):
         # Convert -1 dims in new_shape into positive equivalent.
         if -1 in new_shape:
             breakpoint()
-            data_shape = [dim.value for dim in data.shape.values] 
+            data_shape = [dim.value for dim in data.shape.values]
             total_elements = np.prod(data_shape)
             new_product = 1
             for dim in new_shape:
                 if dim > 0:
                     new_product *= dim
-            
+
             # Replace -1 with positive equivalent
             for i, dim in enumerate(new_shape):
                 if dim == -1:
                     new_shape[i] = int(total_elements / new_product)
-        
+
         breakpoint()
```

Comment:
Debugging leftover

Reply:
Thanks for point this out, this is already in the repo and given Reshape is not working yet I suppose we would remove this breakpoint in a later PR.